pages/_app.tsx

```diff
@@ -3,17 +3,17 @@ import '../styles/global.css';
 import { CacheProvider, EmotionCache } from '@emotion/react';
 import { CssBaseline, ThemeProvider } from '@mui/material';
 import Head from 'next/head';
-import { useRouter } from 'next/router';
-import React, { useEffect } from 'react';
-import TagManager from 'react-gtm-module';
+
+import React from 'react';
+
```

Comment:
@shondriane I thought we had troubleshooted ESLint running for you, can you double check that ESLint is running, and that you are able to see the 2 errors on this page, and then by hitting save, they both auto-correct?